packages/metal-clay-button/src/MetalClayButton.soy

```diff
@@ -18,12 +18,12 @@
 
 
 	{if $href}
-		<a href="{$href}" {call .attributes data="all" /} aria-label="{$aria_description}">
+		<a href="{$href}" {call .attributes data="all" /} aria-label="{$ariaLabel}">
```

Comment:
Hey, @julien. Buttons also can have the aria-label attribute. Could you move it to the `.attributes` template, please?